src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/commands.py

```diff
@@ -120,6 +120,11 @@ def transform_vm(result):
         c.command('list', 'list')
         c.command('delete', 'delete')
 
+with ServiceGroup(__name__, get_devtestlabs_custom_image_operation,
+                  custom_operations) as s:
```

Comment:
You can keep these two lines on the same line. The line length limitation is 120. If it fits in github review panel, it's good.